medusa/providers/nzb/anizb.py

```diff
@@ -118,5 +121,32 @@ def _get_size(self, item):
         """Override the default _get_size to prevent it from extracting using the default tags."""
         return try_int(item.get('size'))
 
+    def _get_episode_search_strings(self, episode, add_string=''):
+        """Get episode search strings."""
+        _ = add_string
```

Comment:
Hm? If you add ```_ = add_string``` you are getting an unused variable, because you are assigning ```_``` and never using it (that's also why you had to ignore it in the test). Or what do you mean?

Since you are overriding the method, you can just override the number of arguments:
```def _get_episode_search_strings(self, episode):```
Will also work as intended.